pkg/web/templates/partials/sunrise/verify_code.html

```diff
@@ -0,0 +1,17 @@
+<div id="verify-code" class="mx-auto max-w-4xl">
```

Comment:
Once a user clicks the submit button in the verify email form, this partial would be loaded into the empty div in verify.html.